deploy/runtime/versions.lk

```diff
@@ -81,86 +87,101 @@ version_upgrade{'2023.9.19'} = define() {
 			value('subarray4_modules_per_string', 1, 'Change default Subarray 4 modules per string from 0 to 1 to avoid simulation error' );
 			value('subarray4_nstrings', 1, 'Change default Subarray 4 strings in parallel from 0 to 1 to avoid simulation error');
 		}
+		
+		//changed module aspect ratio
 		message('The module aspect ratio input is now on the Module page instead of the Shading and Layout page. For the CEC performance model with module library, SAM now calculates the aspect ratio from the length and width data in the module library if it is avalailable.');
+		
+		//DC adjust factor
 		x = oldvalue('dc_adjust');
-		message('DC Losses (availability losses) variable format changed from table to separate variables.');
-		value('dc_adjust_constant', x.constant, 'Convert DC Losses dc_adjust:constant to dc_adjust_constant');
-		value('dc_adjust_en_periods', x.en_periods, 'Convert DC Losses dc_adjust:en_periods to dc_adjust_en_periods');
-		if ( #x.periods < 3 ) { p = [[0,0,0]]; }
-		else { p = x.periods; }
-		value('dc_adjust_periods', p, 'Convert DC Losses dc_adjust:periods to dc_adjust_periods');
-		value('dc_adjust_en_timeindex', x.en_timeindex, 'Convert DC Losses dc_adjust:en_timeindex to dc_adjust_en_timeindex');
-		value('dc_adjust_timeindex', x.timeindex, 'Convert DC Losses dc_adjust:timeindex to dc_adjust_timeindex');
+		message('DC Losses variable format changed from single table ("dc_adjust") to separate variables: dc_adjust_constant (number), dc_adjust_en_timeindex (number), dc_adjust_timeindex (array), dc_adjust_en_periods (number), dc_adjust_periods (array).');
+		value('dc_adjust_constant', x.constant);
+		value('dc_adjust_en_periods', x.en_periods);
+		p = x.periods; //this logic works for both blank period arrays and for completed ones
+		value('dc_adjust_periods', p);
+		value('dc_adjust_en_timeindex', x.en_timeindex);
+		value('dc_adjust_timeindex', x.timeindex);
+		
+		//AC adjust factor
+		x = oldvalue('adjust');
+		message('AC losses variable format changed from single table ("adjust") to separate variables: adjust_constant (number), adjust_en_timeindex (number), adjust_timeindex (array), adjust_en_periods (number), adjust_periods (array).');
+		value('adjust_constant', x.constant);
+		value('adjust_en_periods', x.en_periods);
+		p = x.periods; //this logic works for both blank period arrays and for completed ones
+		value('adjust_periods', p);
```

Comment:
adjust_periods_dc fails on simulate when upgrading the default detailed PV case (without changing anything)

![image](https://github.com/NREL/SAM/assets/54449384/287c62a1-0070-427b-abe0-cedba693d1f8)

Reply:
I can't replicate this on my machine. However, what is getting loaded in seems to have changed and is no longer working for me. Possibly a wxwidgets issue? Let's talk more at our working session...